specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/ContainerApps.json

```diff
@@ -454,6 +513,22 @@
         }
       }
     },
+    "ContainerAppSecret": {
+      "description": "Container App Secret.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Secret Name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "Secret Value.",
+          "type": "string",
+          "readOnly": true
+        }
```

Comment:
Would you need `x-ms-secret` for them?

Reply:
This is only used by a `POST` action to list secrets. The get call has a different contract with `x-ms-secret` set to true. Since we set the `readonly` property to true, we don't need `mutability` property either: [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi013-secrets-must-be-annotated-with-x-ms-secret-and-x-ms-mutability]